base/applications/rapps/unattended.cpp

```diff
@@ -402,6 +445,10 @@ ParseCmdAndExecute(LPWSTR lpCmdLine, BOOL bIsFirstLaunch, int nCmdShow)
     {
         return HandleGenerateInstallerCommand(db, argc - 2, argv + 2);
     }
+    else if (IsStrPrefixI(argv[1], L"rapps:"))
+    {
+        return HandleProtocolCommand(db, wcschr(lpCmdLine + wcslen(argv[0]), ':') + 1, nCmdShow);
```

Comment:
We cannot use argv[1] ?

Reply:
No, I want the unprocessed command line so `rapps://Search/Process Explorer` works.

Reply:
Isn't the link supposed to be URI-encoded in this case? (i.e. `rapps://Search/Process%20Explorer` or `rapps://Search/Process+Explorer`)

Reply:
%20 will work yes.